### DISCUSSION TITLE
Renovate scans the repos irrespective of the schedule present in renovate.json

### DISCUSSION THREAD

**Discussion by sowmya-parsi** [renovatebot/renovate#15299] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

latest

### Please select which platform you are using if self-hosting.

Bitbucket Server

### If you're self-hosting Renovate, tell us what version of the platform you run.

7.21

### Was this something which used to work for you, and then stopped?

I never saw this working

### Describe the bug

Hi Team,

We are trying to have a different schedule for different repos for performing scans. We are using schedule option in renovate.json file in the repository. We observed that even though the repo schedule is different, the renovate scans for dependencies but doesn't create PR, Is this expected behavior? If yes, can you let us know how can we avoid scanning the repos which doesnt come under the schedule time.

Thanks,
Sowmya

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste any log here, between the starting and ending backticks
```

</details>

### Have you created a minimal reproduction repository?

No reproduction repository

**Comment by rarkins** [comment, marked as answer]
This is expected behavior, and not a bug.

Please don't create any more bug reports while you are still new and learning Renovate as you waste our time when we need to convert and close them.
